src/pages/api/arbeidssoker/index.ts

```diff
@@ -30,9 +19,9 @@ async function arbeidssokerStatusHandler(req: NextApiRequest, res: NextApiRespon
   }
 
   const callId = uuid();
-  const url = `${process.env.VEILARBPROXY_URL}?fnr=${payload.pid}&fraOgMed=${formaterDato(
-    new Date()
-  )}`;
+  const formattedDate = formatISO(new Date(), { representation: "date" });
```

Comment:
Kan du bruke komponenten [FormattedDate](https://github.com/navikt/dp-soknadsdialog/blob/main/src/components/FormattedDate.tsx) her?

Reply:
Er det riktig å bruke det i api kontekst? Komponenten returnerer en html element. Kunne vært med naturlig om FormattedDated endres til en hjelpe funksjon som retunerer formattert dato string?

Reply:
Oi, se der ja. Jeg tok det for gitt at det ble brukt i visning her uten å sjekke det noe grundigere. Siden den her trengs som en string i en url så kan du se bort ifra kommentaren min ovenfor 😄